system/jlib/jutil.cpp

```diff
@@ -1647,8 +1651,26 @@ unsigned usTick()
     gettimeofday(&tm,NULL);
     return tm.tv_sec*1000000+tm.tv_usec;
 }
+unsigned __int64 nsTick()
+{
+    if (!use_gettimeofday) {
+        timespec tm;
+        if (clock_gettime(CLOCK_MONOTONIC, &tm)>=0)
+            return ((unsigned __int64)tm.tv_sec)*1000000000+tm.tv_nsec;
```

Comment:
Should we add a 'LL' to the 1000000000 constant ?  Probably not needed since the first term in cast to the proper size.

Reply:
I think this safe, but it would be a sensible change